include/flatbuffers/base.h

```diff
@@ -165,6 +165,10 @@ template<typename T> void WriteScalar(void *p, T t) {
   *reinterpret_cast<T *>(p) = EndianScalar(t);
 }
 
+static inline size_t RoundUp(size_t size, size_t round_size) {
```

Comment:
base.h is shared between FlatBuffers and FlexBuffers. If this is not used by FlexBuffers it doesn't need to be here. In fact, I was expecting it to be just a static class method since it is only used inside one class.